packages/victory-core/src/victory-util/textsize.ts

```diff
@@ -243,6 +246,63 @@ const _approximateTextHeightInternal = (text: string | string[], style) => {
   }, 0);
 };
 
+const _measureWithDOM = (
+  text: string | string[],
+  style?: TextSizeStyleInterface,
+): { width: number; height: number } => {
+  const element = document.createElement("div");
+  element.style.position = "absolute";
```

Comment:
Thoughts on using `fixed` instead of `absolute` just to be _damn sure_ it won't be in the viewport?

Reply:
Talked with Scott, might do the rendering in SVG instead to make double sure it's the correct size (+ RN support)

Reply:
Updated to fixed positioning